docs/FREIGHTER_GUIDE.md

```
@@ -0,0 +1,228 @@
+3. Paste your public key into the **Public Key** field and click **Get test network lumens**.
+
+   Alternatively, fund via the API directly:
+
+   ```bash
+   curl "https://friendbot.stellar.org?addr=YOUR_PUBLIC_KEY"
+   ```
+
+4. Return to Freighter. Your balance should update within a few seconds to show **10,000 XLM**.
+
+> **Screenshot placeholder:** Freighter balance view showing testnet XLM balance after Friendbot funding.
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)